mods/cnc/rules/structures.yaml

```diff
@@ -22,6 +22,10 @@ FACT:
 		IntoActor: mcv
 		Offset: 1,1
 		Facing: 108
+		RequiresCondition: fact-undeploy
+	GrantConditionOnPrerequisite@GLOBALFACTUNDEPLOY:
+		Condition: fact-factundeploy
```

Comment:
I'm not sure about what corrent naming should be but this and 2 lines above don't match, so it wouldn't work and causing Travis to give error.

Reply:
yeah I somehow messed that up